src/components/welcome/WeclomeSection.vue

```diff
@@ -0,0 +1,131 @@
+<script>
+import { defineComponent } from "vue";
+import SectionContainer from "@/components/global/SectionContainer.vue";
+import SectionTitlePrimary from "@/components/global/SectionTitlePrimary.vue";
+import SectionDescription from "@/components/global/SectionDescription.vue";
+import BaseButton from "@/components/global/BaseButton.vue";
+import BaseImage from "@/components/global/BaseImage.vue";
+
+export default defineComponent({
+  name: "WeclomeSection",
```

Comment:
Typo both here and in the file name 🐷 
```suggestion
  name: "WelcomeSection",
```